tensorflow_encrypted/tensor/int32.py

```diff
@@ -111,6 +115,19 @@ def concat(x: List['Int32Tensor'], axis: int) -> 'Int32Tensor':
 
         return Int32Tensor.from_native(tf.concat(backing, axis=axis))
 
+    def binarize(self) -> NativeTensor:
+        """Computes bit decomposition of tensor
```

Comment:
this doesn't seem up to date, eg taking ndarray as input, remove for now?

Reply:
good catch